roles/openshift_node/tasks/storage_plugins/nfs.yml

```diff
@@ -3,30 +3,24 @@
   action: "{{ ansible_pkg_mgr }} name=nfs-utils state=present"
   when: not openshift.common.is_atomic | bool
 
-- name: Check for existence of virt_use_nfs seboolean
-  command: getsebool virt_use_nfs
-  register: virt_use_nfs_output
+- name: Check for existence of {{ item }} seboolean
+  command: getsebool {{ item }}
+  register: getsebool_status
   when: ansible_selinux and ansible_selinux.status == "enabled"
   failed_when: false
   changed_when: false
+  with_items:
+    - virt_use_nfs
+    - virt_sandbox_use_nfs
 
 - name: Set seboolean to allow nfs storage plugin access from containers
   seboolean:
-    name: virt_use_nfs
+    name: "{{ item.item }}"
     state: yes
     persistent: yes
-  when: ansible_selinux and ansible_selinux.status == "enabled" and virt_use_nfs_output.rc == 0
-
-- name: Check for existence of virt_sandbox_use_nfs seboolean (RHEL)
-  command: getsebool virt_sandbox_use_nfs
-  register: virt_sandbox_use_nfs_output
-  when: ansible_distribution != "Fedora" and ansible_selinux and ansible_selinux.status == "enabled"
-  failed_when: false
-  changed_when: false
-
-- name: Set seboolean to allow nfs storage plugin access from containers(sandbox) (RHEL)
-  seboolean:
-    name: virt_sandbox_use_nfs
-    state: yes
-    persistent: yes
-  when: ansible_distribution != "Fedora" and ansible_selinux and ansible_selinux.status == "enabled" and virt_sandbox_use_nfs_output.rc == 0
+  # We need to detect whether or not the boolean is an alias, since `seboolean`
+  # will error if it is an alias.  We do this by inspecting stdout for the boolean name,
+  # since getsebool prints the resolved name.  (At some point Ansible's seboolean module
+  # should learn to deal with aliases)
+  when: ansible_selinux and ansible_selinux.status == "enabled" and item.rc == 0 and item.stdout.find(item.item) != -1
+  with_items: "{{ getsebool_status.results }}"
```

Comment:
Are you sure this is working? I'll try to test on my own but the CI logs look like it's sending garbage to the module.

```
TASK [openshift_node : Set seboolean to allow nfs storage plugin access from containers] ***
changed: [10.8.171.71] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_use_nfs'], u'end': u'2016-10-11 16:03:50.728587', '_ansible_no_log': False, u'stdout': u'virt_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_use_nfs', u'delta': u'0:00:00.012906', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:50.715681', u'warnings': [], 'failed': False})
changed: [10.8.171.63] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_use_nfs'], u'end': u'2016-10-11 16:03:50.805885', '_ansible_no_log': False, u'stdout': u'virt_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_use_nfs', u'delta': u'0:00:00.005430', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:50.800455', u'warnings': [], 'failed': False})
changed: [10.8.171.68] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_use_nfs'], u'end': u'2016-10-11 16:03:50.312613', '_ansible_no_log': False, u'stdout': u'virt_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_use_nfs', u'delta': u'0:00:00.015568', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:50.297045', u'warnings': [], 'failed': False})
changed: [10.8.171.63] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_sandbox_use_nfs'], u'end': u'2016-10-11 16:03:51.320026', '_ansible_no_log': False, u'stdout': u'virt_sandbox_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_sandbox_use_nfs', u'delta': u'0:00:00.002346', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_sandbox_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_sandbox_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:51.317680', u'warnings': [], 'failed': False})
changed: [10.8.171.71] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_sandbox_use_nfs'], u'end': u'2016-10-11 16:03:51.235344', '_ansible_no_log': False, u'stdout': u'virt_sandbox_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_sandbox_use_nfs', u'delta': u'0:00:00.002370', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_sandbox_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_sandbox_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:51.232974', u'warnings': [], 'failed': False})
changed: [10.8.171.68] => (item={'_ansible_parsed': True, u'cmd': [u'getsebool', u'virt_sandbox_use_nfs'], u'end': u'2016-10-11 16:03:50.714343', '_ansible_no_log': False, u'stdout': u'virt_sandbox_use_nfs --> off', '_ansible_item_result': True, u'changed': False, 'item': u'virt_sandbox_use_nfs', u'delta': u'0:00:00.002494', u'stderr': u'', u'rc': 0, 'invocation': {'module_name': u'command', u'module_args': {u'creates': None, u'executable': None, u'_uses_shell': False, u'_raw_params': u'getsebool virt_sandbox_use_nfs', u'removes': None, u'warn': True, u'chdir': None}}, 'stdout_lines': [u'virt_sandbox_use_nfs --> off'], 'failed_when_result': False, u'start': u'2016-10-11 16:03:50.711849', u'warnings': [], 'failed': False})
```